### DISCUSSION TITLE
HM-LC-Dim1TPBU-FM acting as a RGBW Dimmer?

### DISCUSSION THREAD

**Discussion by Stephan-4711** [SukramJ/aiohomematic#1196] [General]
HM-LC-Dim1TPBU-FM is using 0-255 steps for dimming and the developer tools from HA are showing:
supported_color_modes:
  - brightness

this device only acts as dimmer.
For all integration I like to use voice command I need an addional function in node red to change the 0-255 steps to percentage.
Is this a mistake in the device or is this behavior on purpose and if why?

**Comment by SukramJ** [comment]
The implementation is correct. 
See HA dev docs for [light](https://developers.home-assistant.io/docs/core/entity/light#properties)

**Reply by SukramJ** [reply]
Service calls can be used with [percentage](https://www.home-assistant.io/integrations/light/#service-lightturn_on)(brightness_pct).

**Comment by Stephan-4711** [comment]
thanks, that's now working for me.
